### ISSUE TITLE
Zoom to a large number

### ISSUE THREAD

**Issue by renkun-ken** [nx10/httpgd#34]
First, create a simple plot:

```r
plot(rnorm(100))
```

Then keep pressing the zoom-in key, the number could become very large:

<img width="609" alt="image" src="https://user-images.githubusercontent.com/4662568/92935955-12fe5000-f47c-11ea-803c-6ac933f74774.png">

Sometimes it could also cause the following error:

```
> Error: invalid graphics state                                                                                                                                                                    
later: exception occurred while executing callback: 
```

and the graphics no longer responds to any changes.

**Comment by nx10** [comment]
should be fixed with eed6692064ca81a16934b26fb4cbc392eda82bc7

**Comment by renkun-ken** [comment]
Thanks for the quick fix. It works nicely now.
